assets/src/scss/style.scss

```diff
@@ -73,3 +73,17 @@ Text Domain: planet4-master-theme
 @import "vendors/usabilla";
 @import "vendors/gtm";
 @import "~lite-youtube-embed";
+
+// Variables
+:root {
+  --body--font-family: #{$lora};
+  --headings--font-family: #{$roboto};
+  --headings--font-weight: bold;
+  --body--background-color: #{$white};
+  --body--color: #{$grey-80};
+  --link--color: #{$link-color};
+  --link--hover--color: #{$link-color};
+  --link--text-decoration: none;
+  --link--hover--text-decoration: underline;
+  --link--visited--color: #{$link-color-visited};
+}
```

Comment:
Not sure I get this change 🤔

Reply:
This is to make sure that we only initialize them once, when loading this file. The way they were defined before in `_variables.scss` resulted in this being declared multiple times, since that file is imported in several blocks... Which means that they would override the new `style.scss` loaded for the new identity styles! Does it make sense? 😅

Reply:
Yes! Thanks :) And it wouldn't affect the plugin in cases where we use one of these variables? I think we only include `_variables.scss` from the submosule.

Reply:
Hmmm you're right, the color change is currently not applied in the editor 😞 I'll see how to fix it

Reply:
I've refactored this a bit so that now the CSS variables are defined in a new file `_css-variables.scss`, and then we include this file in both the `style.scss` and `editorStyle.scss`. It seems to work as expected, and the new identity styles are also applied to the editor